lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -96,6 +96,7 @@ def call(params)
           ignore_failures = GoodData::Helpers.to_boolean(params.ignore_failures)
           remove_users_from_project = GoodData::Helpers.to_boolean(params.remove_users_from_project)
           do_not_touch_users_that_are_not_mentioned = GoodData::Helpers.to_boolean(params.do_not_touch_users_that_are_not_mentioned)
+          create_non_existing_user_groups = GoodData::Helpers.to_boolean(params.create_non_existing_user_groups || true)
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)